lxc/remote.go

```diff
@@ -191,15 +191,15 @@ func (c *cmdRemoteAdd) runToken(server string, token string, rawToken *api.Certi
 		return errors.New(i18n.G("Failed to add remote"))
 	}
 
-	err = c.addRemoteFromToken(string(line), server, token, rawToken.Fingerprint)
+	err = c.addRemoteFromToken(string(line), server, token, rawToken)
 	if err != nil {
 		return err
 	}
 
 	return nil
 }
 
-func (c *cmdRemoteAdd) addRemoteFromToken(addr string, server string, token string, fingerprint string) error {
+func (c *cmdRemoteAdd) addRemoteFromToken(addr string, server string, token string, rawToken *api.CertificateAddToken) error {
```

Comment:
Can we avoid passing the pointer in here? We dont modify the rawToken right?

Reply:
Yep I'll need to fix this up after the recent changes to `--accept-certificate` usage as well.